app/ui-message/client/message.js

```diff
@@ -172,6 +172,12 @@ Template.message.helpers({
 			return 'false';
 		}
 	},
+	isMultiline() {
```

Comment:
this approach will not work for all cases =/

Reply:
Did some changes via CSS, can you check again?